packages/core/app/styles/navi-core/components/navi-table-sort-icon.less

```diff
@@ -7,14 +7,20 @@
   display: flex;
   flex-flow: column;
   margin-left: 5px;
+  margin-top: -2px;
```

Comment:
Is this to vertically center the icon? If so, can we just use flex box instead since the `.navi-table-sort-icon` is already a flex container.

Reply:
The problem is that the icon itself is not centered, the down arrow is hanging below the center and the up arrow above, centering it won't actually put the arrow in the center.